tests/mochaGlobalSetup.js

```diff
@@ -0,0 +1,10 @@
+const { scraper: { executeScrapeJob }, redis } = require('../routes/instances');
+
+// eslint-disable-next-line
+before(async () => {
+	await redis.flushall();
```

Comment:
really not a good idea to flush redis before trying to scrape all the data, what will you do when you flush and the whole scraping fails? nginx will save your ass because it will serve the cache, but you will have to wait at least 10 minutes until it is updated again, IF the scraping doesnt fail again

Reply:
This is for testing specifically. If scraping fails, tests fail (dy design?)

Reply:
okay okay, i didnt look at the filename, makes sense now haha